addons/mail/static/src/views/fields/statusbar_time/statusbar_time_field.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="web.StatusBarTimeField" t-inherit="web.StatusBarField" t-inherit-mode="primary" owl="1">
```

Comment:
Why prefixing by "web" ? Generally it indicates the source addon :) so "mail.StatusBarTimeField" would say, unless some reasons ?